tests/components/ista_ecotrend/__init__.py

```diff
@@ -0,0 +1 @@
+"""Tests for the ista Ecotrend integration."""
```

Comment:
while the sensor tests are not a hard requirement, a suggest you add them regardless. It's easily done with `snapshot` tests, look at tedee for an example

Reply:
Added snapshot tests for the sensors but still need tests for the statistics import part